### DISCUSSION TITLE
Hue smart button actions

### DISCUSSION THREAD

**Discussion by donutVillan** [Koenkk/zigbee2mqtt#16662] [Devices]
[https://www.zigbee2mqtt.io/devices/8718699693985.html](url) lists 7 actions.
5 of them I’ve managed to create.

- Single press ->
  - `press` -> `release` -> `on`/`off` (toggles between `on` and `off`)
- Double press ->
  - `press` -> `release` -> `press` -> `release` -> `on`/`off` (toggles between `on` and `off`)
- Press and hold ->
  - `press` -> `hold` -> `hold` -> … -> `release`

But how do I get the actions `skip_backward` and `skip_forward`?

**Comment by FrankHogervorst** [comment]
I wonder the same thing! Did you find out?
